src/main/java/de/learnlib/ralib/tools/AbstractToolWithRandomWalk.java

```diff
@@ -217,7 +216,7 @@ public void setup(Configuration config) throws ConfigurationException {
         EnumSet<Category> cat = OPTION_LOGGING_CATEGORY.parse(config);
         for (Handler h : root.getHandlers()) {
             h.setLevel(lvl);
-            h.setFilter(new CategoryFilter(cat));
+            // h.setFilter(new CategoryFilter(cat));	// TODO: Change to use new learnlib API
         }
```

Comment:
I think it would be cleanest to just completely remove this logger configuration. With the switch to Slf4J, the choice of the logging framework (including its configuration) is delegated to the user-space (see, e.g., [this example](https://github.com/LearnLib/learnlib/blob/develop/examples/src/main/resources/logback.xml)). So any magic from us just seems counter-productive.

I think early versions of AutomataLib and LearnLib were a little bit too over-ambitious in some regards by trying to re-invent the wheel.

Reply:
Yes, I agree with this comment / suggestion.  I will go ahead and remove this code.

Reply:
I locally removed more code that also removes the logging configuration options. Should we include this in this PR?